src/pay/types/charge.ts

```diff
@@ -0,0 +1,120 @@
+export type CommerceCharge = {
```

Comment:
Dope!

For all types, just add them in `src/pay/types.ts`.

Reply:
Also do not add types that are not used yet.

It will help us have better code review of adding types only of things used.